resources/definitions/winbo_mini2.def.json

```diff
@@ -0,0 +1,137 @@
+{
+    "version": 2,
+    "name": "Winbo FDM-Mini2",
```

Comment:
I forgot this in the previous review. We are not allowing to use **FDM** in the printer names. You can choose for instance **FFF** standing for Fused Filament Fabrication. We recently change our **Custom FDM Printer** to be **Custom FFF Printer**. Of course you can also choose another different name that doesn't contain the acronym.

Reply:
That's what they name their printers! You can't require them to rename their products, can you?
https://www.winbo.top/3d-printers/

Reply:
FDM is trademarked by Stratasys though, that's why we had to change it, courtesy of our legal council....

Reply:
The filename is also without fdm, so I guess Winbo Mini2 would do?

Reply:
@fieldOfView I know that I can't require to change their name. Of course I'm not saying that they must change the product name. However now we can't accept those names due to legal reasons. If they can just change how the product shows in Cura, then it's fine. Otherwise, we can't accept this PR.